### ISSUE TITLE
[QUESTION] Default response_model on include_router

### ISSUE THREAD

**Issue by zd4y** [tiangolo/fastapi#1464]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I put a default `response_model` when including a router? For example, having this router:

```python
# routers/users.py

router = APIRouter()

@router.get('/{user_id}', response_model=User)
def get_user(user_id: int, db: Session = Depends(get_db)):
    return crud.get_user(db=db, user_id=user_id)


@router.put('/{user_id}', response_model=User)
def update_user(user_id: int, user: UserUpdate, db: Session = Depends(get_db)):
    return crud.update_user(db=db, user_id=user_id, user=user)
```

I am using the same `response_model` in all routes, so I might rather put it when including the router:

```python
# main.py

app.include_router(users.router, prefix='/users', default_response_model=User)
```

But this gives an error: `TypeError: include_router() got an unexpected keyword argument 'default_response_model'` for both `default_response_model` and `response_model`.

Is there any reason why this is not supported?

**Comment by dbanty** [comment]
I believe you can put the response_model in the declaration of `APIRouter()` instead of where you register it.

```python
router = APIRouter(response_model=User)
```

Not at a computer to test that on, but hopefully it does the trick 😅

**Comment by zd4y** [comment]
Same error: `TypeError: __init__() got an unexpected keyword argument 'response_model'`

**Comment by dbanty** [comment]
Yeah that's my bad, I really thought this was already a feature 😨.  Seems like a good feature to implement for sure.

**Comment by mpaccione** [comment]
I have the same problem! All of my endpoints return JSON (except for a string for some dev only apis). Used the router to have logical seperation of concerns but it seems like I can't find very good documentation on this issue?
